src/version.h.in

```diff
@@ -21,4 +21,5 @@
 #pragma once
 
 #define VERSION "@PROJECT_VERSION@"
+#define REDIS_VERSION "4.0.0"
```

Comment:
I think it should not be in `version.h.in`, which is a header for kvrocks version configuration.

You can put them just in `server.h`.

Reply:
Yes, twice is right here, I didn't notice this.